chef/automate
# Fixed bug "waived control not being listed"

### :nut_and_bolt: Description: What code changed, and why?
A bug was reported stating that an Automate user was not able to see the waived controls. When viewing a node under the "Compliance" section of Chef Automate, clicking on "Waived controls" shows an incomplete list of all controls, expected behaviour is to only show waived controls.



### :chains: Related Resources
The bug ticket: https://chefio.atlassian.net/browse/STALWART-346

Video:
https://user-images.githubusercontent.com/36985548/221857326-f51dea13-bca7-4e94-9496-7e43757b7463.mov



### :+1: Definition of Done

- Get waived controls

### :athletic_shoe: How to Build and Test the Change
rebuild compliance service
- load compliance data
- Navigate to Compliance-> Reports → Nodes 
- Click on any node and then click on Waived Nodes
- You should be able to see waived nodes

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
